config/jobs/kubernetes-sigs/kube-storage-version-migrator/kube-storage-version-migrator-presubmits.yaml

```diff
@@ -74,3 +74,44 @@ presubmits:
     annotations:
       testgrid-dashboards: sig-api-machinery-kube-storage-version-migrator
       testgrid-tab-name: pr-e2e-fully-automated
+  - name: pull-kube-storage-version-migrator-fully-automated-e2e-optional-custom-build
+    decorate: true
+    extra_refs:
+    - org: roycaihw
+      repo: kubernetes
+      base_ref: storage-version/conditions
+      path_alias: k8s.io/kubernetes
+    optional: true
+    always_run: true
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+      preset-dind-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201111-a263fd7-master
+        command:
+        - runner.sh
+        - /workspace/scenarios/kubernetes_e2e.py
+        args:
+        - --up
+        - --down
+        - --build=quick
+        - --check-leaked-resources
+        - --gcp-master-image=gci
+        - --gcp-node-image=gci
+        - --gcp-nodes=1
+        - --gcp-zone=us-central1-f
+        - --provider=gce
+        - --test=false
+        - --test-cmd=../test/e2e/test-fully-automated.sh
+        - --test-cmd-args=--skip=\[Disruptive\]
+        - --timeout=50m
+        - --runtime-config=internal.apiserver.k8s.io/v1alpha1=true
+        - --env=KUBE_FEATURE_GATES=StorageVersionAPI=true,APIServerIdentity=true
```

Comment:
Are these flags the only needed modifications? Do we need to specify the k8s version somewhere? If not, can we reuse the `pull-kube-storage-version-migrator-fully-automated-e2e`?

Reply:
The k8s version is specified in `extra_refs`